src/main/java/gololang/Predefined.java

```diff
@@ -598,7 +603,7 @@ public static boolean isArray(Object object) {
    * @return the Character value.
    * @throws IllegalArgumentException if {@code obj} is not a number or a String.
    */
-  public static Object charValue(Object obj) throws IllegalArgumentException {
+  public static Object charValue(Object obj) {
```

Comment:
> Not mandatory since it's an unchecked exception (pmd was complaining). But it can indeed be useful as a documentation purpose

In that case PMD is wrong :smile:

(note that I don’t like PMD)